app-games/src/main/java/com/aptoide/android/aptoidegames/play_and_earn/presentation/home/PaEHeaderBundle.kt

```diff
@@ -55,7 +55,7 @@ fun PaEHeaderBundle(
           app = app,
           onClick = {
             navigate(
-              buildAppViewRoute(app).withItemPosition(index)
+              buildAppViewRoute(app, isGamified = true).withItemPosition(index)
```

Comment:
should this be only passed when we navigate through here and the large item card ? Or should there be another way to get it so it would be shown even in "normal flows" ?